src/home/CashInBottomSheet.tsx

```diff
@@ -114,7 +118,7 @@ function CashInBottomSheet() {
     <Modal
       animationIn="slideInUp"
       animationInTiming={800}
-      isVisible={isModalVisible}
+      isVisible={isModalVisible && !inSanctionedCountry}
```

Comment:
Ah this feels a bit hacky because the only screen should be `SanctionedCountryErrorScreen`.
I guess the previous hacky was a bit nicer to avoid this (and future similar issues), since we were not rendering the navigator 😄

Reply:
Moved this logic to `WalletHome.tsx` which has `shouldShowCashInBottomSheet` with some other conditional display logic.